apps/prairielearn/src/lib/bot-grading.sql

```diff
@@ -0,0 +1,31 @@
+-- BLOCK select_instance_questions_manual_grading
+SELECT
+  iq.*,
+  COALESCE(g.name, u.name) AS user_or_group_name
+FROM
+  instance_questions AS iq
+  JOIN assessment_instances AS ai ON (ai.id = iq.assessment_instance_id)
+  LEFT JOIN users AS u ON (u.user_id = ai.user_id)
+  LEFT JOIN groups AS g ON (g.id = ai.group_id)
+WHERE
+  ai.assessment_id = $assessment_id
+  AND iq.assessment_question_id = $assessment_question_id
+  AND iq.status != 'unanswered'
```

Comment:
You are selecting all instance questions, including those that are already graded (`requires_manual_grading = false`). Is this on purpose?

Reply:
@Victorsss-Orz did you fix this?

Reply:
No I have not modified this yet.

Reply:
Sorry, but I don't agree with that approach. If an instructor tags a few questions as requiring grading while others are left graded, then you should not grade the other questions. The "graded" questions might be ones that the instructor has an easy way to grade (e.g., they want to manually grade only questions that the autograder is unable to catch), or that they have already done, and you don't want to override that. Note that this is as easy as adding `AND iq.requires_manual_grading` to your query.

Reply:
I agree with @jonatanschroeder, we should do as he says here.

Reply:
Ok. I'll change them